content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
+The Cosign command requests a certificate from our Certificate Authority Fulcio. Fulcio checks your identity by using an authentication protocol (OpenID Connect) to look at your email address. If your identity is correct, Fulcio grants a short-lived, time-stamped certificate. The certificate is bound to the public key to attest to your identity.  This activity is logged using Rekor.
+ 
+Note that you don’t need to use a key to sign.  Currently, you can authenticate with Google, GitHub, or Microsoft. For more information, read [Keyless Signatures](https://docs.sigstore.dev/cosign/keyless/).
 
-Any valid payloads are printed to `stdout`, in JSON format.
-Note that these signed payloads include the digest of the container image, which is how we can be
-sure these "detached" signatures cover the correct image.
+Cosign has additional options and features.   For more information enter the command:
 
-```shell
-$ cosign verify --key cosign.pub dlorenc/demo
-The following checks were performed on these signatures:
-  - The cosign claims were validated
-  - The signatures were verified against the specified public key
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8"},"Type":"cosign container image signature"},"Optional":null}
+```
+cosign sign-blob --help
 ```
 
-## Kubernetes Integrations
+### Verifying a signed blob
 
-Cosign comes with a few built-in Kubernetes integrations: `Secret` generation, and a [policy webhook](/policy-controller/overview/) `policy-controller`.
-In addition to the `policy-controller`, Cosign is also compatible with and supported by other policy engines such as:
+To verify a signed blob, you need to provide three pieces of information:
+* The certificate, found in the bundle
+* The signature, also found in the bundle
+* The identity used in signing
 
-* [Conaisseur](https://github.com/sse-secure-systems/connaisseur#what-is-connaisseur)
-* [Kyverno](https://kyverno.io/docs/writing-policies/verify-images/)
-* [OPA Gatekeeper](https://github.com/sigstore/cosign-gatekeeper-provider)
+The following example verifies the signature on file.txt from user "name@example.com" issued by "accounts@example.com":
 
-To learn how to use Cosign with Kubernetes, review [Kubernetes](/cosign/kubernetes/).
+```
+$ cosign verify-blob file.txt --bundle txtbundle.bundle --certificate-identity=name@example.com 
+                              --certificate-oidc-issuer=https://accounts.example.com
+```
 
-## More Info
+To verify, Cosign queries the transparency log to compare the public key to what’s in Rekor, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by OpenID Connect were valid.
```

Comment:
I thinking saying "queries the transparency log" and "Rekor" like this will be confusing by implying they're different things. Also, a user who lands on the Verify instructions might not have read the Signing instructions, so they haven't been introduced to the names yet.

Reply:
+1

nit, the certificate is issued by the "certificate authority", not "OpenID Connect".

Reply:
reworded so that Rekor is mentioned as the location of the transparency log and changed openid connect to certificate authority.